docs/Onboarding/Prerequisites_Guide.md

```diff
@@ -35,34 +38,45 @@ You will be working through and updating the following information in the prereq
 9. Documented Tests
 
 ## Resources
+
 - Identify engineers that are assigned to work on & hold shared responsibility for product onboarding, test execution and ongoing scenario maintenance. See the [Scenario Maintenance Policy](../Policy/Maintenance/Scenario_Maintenance_Policy.md) and this [RACI Chart](RACI_Chart.md) to understand the roles & responsibilities for this onboarding.
 
 ## Upstream Considerations
+
 Running in OpenShift CI is by default upstream and open to the public. All generated output and artifacts will be made public when run as a prow job. This means that the following MUST be true about your layered product deployment and test automation
```

Comment:
```suggestion
Running in OpenShift CI is by default upstream and open to the public. All generated output and artifacts will be made public when run as a Prow job. This means that the following MUST be true about your layered product deployment and test automation:
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)